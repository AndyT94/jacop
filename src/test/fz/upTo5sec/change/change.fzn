array [1..5] of var int: x:: output_array([1..5]) = [4,4,3,4,1];
array [1..5] of var int: y:: output_array([1..5]) = [1,2,4,3,7];
solve  satisfy;
