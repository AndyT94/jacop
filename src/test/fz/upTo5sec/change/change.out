x = array1d(1..5, [4, 4, 3, 4, 1]);
y = array1d(1..5, [1, 2, 4, 3, 7]);
----------